core/frontend/helpers/get.js

```diff
@@ -153,6 +153,7 @@ module.exports = function get(resource, options) {
 
     // Parse the options we're going to pass to the API
     apiOptions = parseOptions(ghostGlobals, this, apiOptions);
+    apiOptions.context = {member: data.member};
```

Comment:
Is it correct/safe to always override context here, or should we be merging with an existing `context`?